### PR TITLE
Linux find cmd examples

### PR DESCRIPTION
**Pull request by godchaser**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/english/linux/using-the-find-command/index.md [hunk 1]
```diff
@@ -27,3 +27,30 @@ Other parameters are optional and can be combined in any ways you find useful:
 * The name parameter lets you specify what you want to find by name, either with a literal string ('filename.txt') or using wildcards ('file?.*').
 
 `man find` will show you many more parameters, and is worth reviewing. Find can locate files by name, user, creation date, size and much more. Next time you're looking for something, find it!
+
+# Useful find commands
```
Comment (thecodingaviator) [inline guide/english/linux/using-the-find-command/index.md]:
```suggestion
## Useful find commands
```

File: guide/english/linux/using-the-find-command/index.md [hunk 2]
```diff
@@ -27,3 +27,30 @@ Other parameters are optional and can be combined in any ways you find useful:
...
+## Find all javascript files
```
Comment (thecodingaviator) [inline guide/english/linux/using-the-find-command/index.md]:
```suggestion
### Find all javascript files
```

File: guide/english/linux/using-the-find-command/index.md [hunk 3]
```diff
@@ -27,3 +27,30 @@ Other parameters are optional and can be combined in any ways you find useful:
...
+```bash
+find . -type f -name "*.js"
+./server.php
+./model.js
+./config.js
+```
+
+## Find and remove multiple files
```
Comment (thecodingaviator) [inline guide/english/linux/using-the-find-command/index.md]:
```suggestion
### Find and remove multiple files
```

File: guide/english/linux/using-the-find-command/index.md [hunk 4]
```diff
@@ -27,3 +27,30 @@ Other parameters are optional and can be combined in any ways you find useful:
...
+To find and remove multiple files such as .txt, then use.
+```bash 
+find . -type f -name "*.txt" -exec rm -f {} \;
+```
+
+## Find last 30 days modified files
```
Comment (thecodingaviator) [inline guide/english/linux/using-the-find-command/index.md]:
```suggestion
### Find last 30 days modified files
```

File: guide/english/linux/using-the-find-command/index.md [hunk 5]
```diff
@@ -27,3 +27,30 @@ Other parameters are optional and can be combined in any ways you find useful:
...
+To find all the files which are modified 30 days back.
+```bash
+# find / -mtime 30
+```
+
+## Find files with size between 100MB – 200MB
```
Comment (thecodingaviator) [inline guide/english/linux/using-the-find-command/index.md]:
```suggestion
### Find files with size between 100MB – 200MB
```

**Review by thecodingaviator — CHANGES_REQUESTED** [review]

**Comment by RandellDawson** [comment]
@thecodingaviator I made your suggested changes on behalf of the user.

**Review by thecodingaviator — APPROVED** [review]
LGTM

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**